src/analysis/normed/order/lattice.lean

```diff
@@ -31,30 +31,42 @@ normed, lattice, ordered, group
 -/
 
 /-!
-### Normed lattice orderd groups
+### Normed lattice ordered groups
 
 Motivated by the theory of Banach Lattices, this section introduces normed lattice ordered groups.
 -/
 
 local notation (name := abs) `|`a`|` := abs a
 
+section solid_norm
+
+/-- Let `α` be an `add_comm_group` with a `lattice` structure. A norm on `α` is solid if the balls
+of `α` for this norm are solid sets. -/
```

Comment:
The Lean version of this docstring hasn't really been formalized, right? Afaict, there is no connection yet between `has_solid_norm` and `is_solid`.
Could you please add the obvious statements?

Reply:
Oh yes, somehow I squeezed out the result that Yaël suggested. I put it back. 

I still have quite a some questions about the right way to do things:
- I defined `is_solid` for `add_comm_group` since it is used only by [normed_lattice_add_comm_group](https://leanprover-community.github.io/mathlib_docs/analysis/normed/order/lattice.html#normed_lattice_add_comm_group) but it could be defined for `comm_group` and then use `@[to_additive]`. Is it worth it? 
- At the moment, `is_solid_ball` lives in the root namespace. I could move it to the `lattice_ordered_add_comm_group` (with `is_solid`), but does that makes sense when we have [dual_solid](https://leanprover-community.github.io/mathlib_docs/analysis/normed/order/lattice.html#dual_solid) living also in the root namespace? And in fact maybe `is_solid_ball` should be proved only for `normed_lattice_add_comm_group`?

I think at this point, I'd like to get some feedback from somebody that know more about this topic than me...

Reply:
Yeah, we're missing an expert here.

Reply:
@mans0954, can you have a look at this PR and let us know what you think about it?